b1ddi/data_source_dns_auth_nsg.go

```diff
@@ -27,6 +27,11 @@ func dataSourceConfigAuthNSG() *schema.Resource {
 				Elem:        dataSourceSchemaFromResource(resourceConfigAuthNSG),
 				Description: "List of DNS Auth NSGs matching filters. The schema of each element is identical to the b1ddi_dns_auth_nsg resource schema.",
 			},
+			"tags": {
+				Type:        schema.TypeMap,
+				Optional:    true,
+				Description: "Configure a map of tag filters to be applied on the search result.",
```

Comment:
Can we be consistent and use the same `Description` field for all types?

Reply:
resolved